utils/appliance/__init__.py

```diff
@@ -1088,6 +1089,24 @@ def deploy_merkyl(self, start=False, log_callback=None):
             log_callback("Setting it to start after reboot")
             client.run_command("chkconfig merkyl on")
 
+    @logger_wrap("Adding ca certs: {}")
+    def add_ca_certs(self, log_callback=None):
```

Comment:
**REQUIRED**
This method deploys only one certificate and has a name `add_ca_certs`. Please add `certificates` argument and rewrite the code.